Assets/Scripts/Data Model/EntityType.cs

```diff
@@ -1,226 +1,229 @@
-﻿using UnityEngine;
+﻿using System;
 using System.Collections.Generic;
-using System;
+using UnityEngine;
 
-public class EntityType
+namespace MSP2050.Scripts
 {
-	public string Name { get; set; }
-
-	public SubEntityDrawSettings DrawSettings { get; set; }
+	public class EntityType
+	{
+		public string Name { get; set; }
 
-	public long capacity = 1; //Per km2 for areas
-	public float investmentCost = 1f;
-	public string description = "temporary text";
-	public int availabilityDate = -10;
-	public int value = 0; // In EEZLayers these are country ID's
-	public string media;
-	public EApprovalType requiredApproval = EApprovalType.NotDependent;
+		public SubEntityDrawSettings DrawSettings { get; set; }
 
-	//public Dictionary<EnergyKPI.PressureType, float> pressureValues = new Dictionary<EnergyKPI.PressureType, float>(){
-	//    { EnergyKPI.PressureType.ArtificialSubstrate, 1f },
-	//    { EnergyKPI.PressureType.BottomDisturbance, 1f },
-	//    { EnergyKPI.PressureType.Noise, 1f },
-	//    { EnergyKPI.PressureType.SurfaceDisturbance, 1f }};
+		public long capacity = 1; //Per km2 for areas
+		public float investmentCost = 1f;
+		public string description = "temporary text";
+		public int availabilityDate = -10;
+		public int value = 0; // In EEZLayers these are country ID's
+		public string media;
+		public EApprovalType requiredApproval = EApprovalType.NotDependent;
 
-	public EntityType()
-	{
-		Name = "entity type";
-		DrawSettings = new SubEntityDrawSettings();
-	}
+		//public Dictionary<EnergyKPI.PressureType, float> pressureValues = new Dictionary<EnergyKPI.PressureType, float>(){
```

Comment:
Commented code, as always see if it makes sense to leave it there for the future, otherwise remove it! :)